xain_fl/grpc/test_coordinator.py

```diff
@@ -29,38 +26,61 @@ def test_rendezvous_accept():
     assert result.response == RendezvousResponse.ACCEPT
 
 
-def test_rendezvous_later():
-    coordinator = Coordinator(required_participants=1)
+def test_rendezvous_later_fraction_1():
+    coordinator = Coordinator(
+        minimum_participants_in_round=1, fraction_of_participants=1.0
+    )
     coordinator.on_message(coordinator_pb2.RendezvousRequest(), "participant1")
     result = coordinator.on_message(coordinator_pb2.RendezvousRequest(), "participant2")
 
     assert isinstance(result, coordinator_pb2.RendezvousReply)
     assert result.response == coordinator_pb2.RendezvousResponse.LATER
 
 
+def test_rendezvous_later_fraction_05():
+    coordinator = Coordinator(
+        minimum_participants_in_round=1, fraction_of_participants=0.5
+    )
+
+    # with 0.5 fraction it needs to accept at least two participants
+    coordinator.on_message(coordinator_pb2.RendezvousRequest(), "participant1")
+    result = coordinator.on_message(coordinator_pb2.RendezvousRequest(), "participant2")
+
```

Comment:
for completeness you may as well check `participant1` as well

Reply:
thanks. added